cmd/podman/shared/create.go

```diff
@@ -282,13 +282,26 @@ func configurePod(c *GenericCLIResults, runtime *libpod.Runtime, namespaces map[
 	if err != nil {
 		return namespaces, err
 	}
+	hasUserns := false
+	if podInfraID != "" {
+		podCtr, err := runtime.LookupContainer(podInfraID)
```

Comment:
Don't need a lookup here - it's guaranteed to be full ID, so we can do a straight `runtime.Container()`

Reply:
thanks.  Pushed a new version